.github/workflows/check-semver.yml

```
@@ -22,24 +27,46 @@ jobs:
       - name: Rust compilation prerequisites
         run: |
           rustup default nightly-2024-03-01
-          rustup target add wasm32-unknown-unknown --toolchain nightly-2024-03-01
           rustup component add rust-src --toolchain nightly-2024-03-01
 
       - name: install parity-publish
-        run: cargo install parity-publish@0.5.1
+        run: CARGO_TARGET_DIR=./target/ cargo install parity-publish -q --git https://github.com/paritytech/parity-publish --branch oty-max-allowed-bump
```

```suggestion
        run: CARGO_TARGET_DIR=./target/ cargo install parity-publish@0.6.0 -q
```
Keeping this open until its released.

---

Hey Oli, just in case you've missed it, the 0.6.0 version was [released](https://crates.io/crates/parity-publish/0.6.0)